matching-notification-resources/src/main/resources/resources/uicomponents/matchingNotification/matchesTable.js

```diff
@@ -120,6 +129,8 @@ var PhenoTips = (function (PhenoTips) {
         $$('th[data-column="foundTimestamp"]')[0] && $$('th[data-column="foundTimestamp"]')[0].on('click', function(event) {this._sortByColumn('foundTimestamp');}.bind(this));
 
         Event.observe(window, 'resize', this._buildTable.bind(this));
+
+        Event.observe(document, "matches:refreshed", this._showMatches.bind(this));
```

Comment:
just curious what is different on patient page that we need this even handler now, but did not need it in the past?